### ISSUE TITLE
No longer wish to use GratiPay: what's to do with personal account and OSS project org balances?

### ISSUE THREAD

**Issue by ribasushi** [gratipay/gratipay.com#3587]
Hiya!

I couldn't find anything in the tickets fitting my situation, so figured I will open a new ticket. There are two accounts that I opened back in the day when I was excited about (back then) gittip.

* https://gratipay.com/~ribasushi/
* https://gratipay.com/~DBIx-Class/

At present I feel that the mission of gratipay diverged sufficiently from my own values, to consider ending use of the service for both my personal account and the project. I would like to close both accounts, without reinstating any further "paydays" for them, and convert the available balances as a donation to the [EPO](http://www.enlightenedperl.org/donations.html).

What is the best way forward on this? I am not particularly fussed about going through whatever verification process, but given that I do not intend to use the accounts anymore, this seems like the wrong step to take.

Cheers!

P.S. Note that in the case of the DBIx::Class team the intention to move all funds to the EPO was put in place at the time of account creation, and never changed since.

**Comment by whit537** [comment]
Hey @ribasushi, if you connect a PayPal account to the two user accounts I will be happy to white-list the accounts for a one-time payout. Once that is done you can close the accounts once we bring back account closing (#3461), which is a high priority. I've added a +1 for you to #3461.

However, you can only connect an email (including PayPal emails) to one account at a time. I've whitelisted both for payout, so I would suggest doing one this week and the other next week. Let's leave this open until you're paid out, and then pick up with #3461. Sound good?

**Comment by chfoo** [comment]
I'm not sure if I should open another issue, but I'm in the same situation. I would also like to have my account paid out which is https://gratipay.com/~chfoo/. I have a PayPal account already added. I sent an email support request (ticket 2462) not realizing that the GitHub issues wasn't just for code. Sorry if I'm being impatient. Thanks.

**Comment by ribasushi** [comment]
Hi @whit537!

I have connected both accounts to different emails (less friction this way). I am not sure how/who can verify `paypal@enlightenedperl.org` (the official donation address [as listed here](http://www.enlightenedperl.org/donations.html)). Is that step at all necessary in this case?

**Comment by ribasushi** [comment]
@whit537 I got in touch with the EPO people, a confirmation won't work - they already have an account using this very email. What other options are there?
/cc @shadowcat-mdk

**Comment by mattbk** [comment]
+1 from Freshdesk Ticket #2523.

**Comment by whit537** [comment]
@ribasushi Looks like we successfully paid out [~ribasushi](https://gratipay.com/~ribasushi/) last week. Once we land #3601 you'll be able to close that account. For the [~DBIx-Class](https://gratipay.com/~DBIx-Class/) account, can you withdraw to the same PayPal email you used for ~ribasushi, and then forward the funds from there to `paypal@enlightenedperl.org`?

**Comment by whit537** [comment]
@chfoo I've whitelisted your account for a payout, watch for that next week. Sorry for not getting to [FD2462](https://gratipay.freshdesk.com/helpdesk/tickets/2462) sooner. We're behind on support. :-(

**Comment by whit537** [comment]
@mattbk Let's keep tickets such as [FD2523](https://gratipay.freshdesk.com/helpdesk/tickets/2523) in the FD queue and not copy them over here. Let's close this out once @ribasushi and @chfoo are taken care of.

**Comment by ribasushi** [comment]
> Hey @ribasushi, if you connect a PayPal account to the two user accounts I will be happy to white-list the accounts for a one-time payout. 

Ok. The [second account]() now has a connected paypal, as per the workaround suggested in https://github.com/gratipay/gratipay.com/issues/3604#issuecomment-120389617. Please whitelist when time permits.

Cheers!

**Comment by whit537** [comment]
Thanks @ribasushi, whitelisted. Watch for a payout on that one next week!

**Comment by ribasushi** [comment]
Both accounts have now been zeroed. Thanks!

 I am not sure how to proceed wrt marking both accounts as "do not transfer to the 2.0 platform", so not closing ticket yet.

Thoughts?

**Comment by whit537** [comment]
@ribasushi Have you tried going through the "Close Account" workflow at the bottom of [your settings page](https://gratipay.com/about/me/settings/)? Is that what you're after?

**Comment by ribasushi** [comment]
@whit537 I didn't realize the `account closing` functionality came back. We are all set here.

Thanks a lot!

**Comment by whit537** [comment]
Sorry, yeah, brought it back late last week in #3601. Cheers, @ribasushi! :-)
